network-mux/src/Network/Mux/Timeout.hs

```diff
@@ -94,7 +94,7 @@ withTimeoutSerialNative body = body MonadTimer.timeout
 --
 -- The implementation here only relies upon 'threadDelay' which has an
 -- efficient implementation on all platforms. It uses a separate monitoring
--- thread which will through an exception to terminate the action if the
+-- thread which will throw an exception to terminate the action if the
```

Comment:
This is unrelated to the PR, create a separate commit (a separate PR is not needed) for it.